usecases/vessel-detection/index.md

```diff
@@ -0,0 +1,211 @@
+# Vessel Detection
+
+In this notebook we will learn how to apply adaptive thresholding to SENTINEL1_GRD data within OpenEO Platform. We will then take a look at our results against vessel location data from the Maritime Traffic Agency over the Adriatic. We will be using the the openeo-python-client to prepare our process graph, and a Plotly Dash dashboard to interact with our results.
+
+### Recap
+
+We've been through a number of iterations to arrive at this process graph.
+
+1. Recieved an initial implementation from Planetek, which we packaged and released as a custom function, "vessel_detection". 
+2. Porting to Xarray. This resulted in a single function called "adaptive_thresholding", that removed the dependency to the ellipsoid corrected SENTINEL1_GRD imagery.
+3. Porting to existing OpenEO processes. This addresses the feedback from the previous review, i.e. the implementation should be reproducable for this use case.
+
+These iteration steps roughly describe the process of how we can go about onboarding new processes into the OpenEO Platform.
```

Comment:
I wouldn't turn this into a general recommendation, a major feature is that no docker or xarray knowledge is required, and when reading on, the algorithm is not too difficult to port into openEO in a single step.

If we want to recount how things went in the project, I would do that in a separate section at the end, as its less relevant for a general audience.

Reply:
Yeah, this is more intended in a recount of how it went, so I'll restructure and move it to the end.

Reply:
ok